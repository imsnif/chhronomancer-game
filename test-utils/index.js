module.exports = {
  mockFB () {
    const login = require('react-native-facebook-login')
    const FBLogin = jest.fn()
    FBLogin.mockReturnValue(null)
    login.FBLogin = FBLogin
    login.FBLoginManager = {LoginBehaviors: {Web: 2}}
    return FBLogin
  },
  navigatorMock (routeName) {
    return {
      getCurrentRoutes: () => [routeName],
      push: () => {}
    }
  }
}
