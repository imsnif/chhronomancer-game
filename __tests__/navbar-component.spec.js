import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'
import serverMock from '../server-mock'
import { navigatorMock } from '../test-utils'
import statsStore from '../stores/stats'

jest.mock('mobx-react/native', () => require('mobx-react/custom'))

beforeEach(() => {
  jest.useFakeTimers()
  serverMock()
})

afterEach(() => {
  const clockStore = require('../stores/clock').default
  jest.useRealTimers()
  clockStore.destroy()
  jest.useFakeTimers()
  statsStore.clearWinner()
})

test('Navbar (character-sheet) => renders correctly', () => {
  const NavBar = require('../components/navbar').default
  const tree = renderer.create(
    <NavBar navigator={navigatorMock({screenName: 'character-sheet'})} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('Navbar (timeline-grid) => renders correctly', () => {
  const NavBar = require('../components/navbar').default
  const tree = renderer.create(
    <NavBar navigator={navigatorMock({screenName: 'timeline-grid'})} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('Navbar (timeline-grid) => renders correctly', () => {
  const NavBar = require('../components/navbar').default
  const tree = renderer.create(
    <NavBar navigator={navigatorMock({screenName: 'feed'})} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('Navbar (different route) => renders correctly', () => {
  const NavBar = require('../components/navbar').default
  const tree = renderer.create(
    <NavBar navigator={navigatorMock({screenName: 'foo'})} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('Navbar (winner declared) => renders correctly', () => {
  statsStore.announceWinner(1)
  const NavBar = require('../components/navbar').default
  const tree = renderer.create(
    <NavBar navigator={navigatorMock({screenName: 'foo'})} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
