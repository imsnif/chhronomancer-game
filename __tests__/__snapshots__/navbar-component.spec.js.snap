// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar (character-sheet) => renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#141313",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column",
        "height": 45,
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#329932",
          "flex": 0,
          "fontFamily": "telegrama_raw",
          "height": 19,
          "includeFontPadding": false,
          "lineHeight": 22,
          "textAlign": "center",
          "textAlignVertical": "bottom",
        }
      }
    >
      Actions: 
      10
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
          "height": 15,
          "justifyContent": "space-between",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          [ Character ]
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          Timelines
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          Feed (2)
        </Text>
      </View>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#329932",
          "fontSize": 45.911331934501746,
          "height": 40.019999999999996,
          "includeFontPadding": false,
          "lineHeight": 15,
          "textAlignVertical": "top",
        }
      }
    >
      +--------------------------------------------------------+
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 2,
      }
    }
  />
</View>
`;

exports[`Navbar (different route) => renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#141313",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column",
        "height": 45,
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#329932",
          "flex": 0,
          "fontFamily": "telegrama_raw",
          "height": 19,
          "includeFontPadding": false,
          "lineHeight": 22,
          "textAlign": "center",
          "textAlignVertical": "bottom",
        }
      }
    >
      Actions: 
      10
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
          "height": 15,
          "justifyContent": "space-between",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          Character
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          Timelines
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          Feed (2)
        </Text>
      </View>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#329932",
          "fontSize": 45.911331934501746,
          "height": 40.019999999999996,
          "includeFontPadding": false,
          "lineHeight": 15,
          "textAlignVertical": "top",
        }
      }
    >
      +--------------------------------------------------------+
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 2,
      }
    }
  />
</View>
`;

exports[`Navbar (timeline-grid) => renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#141313",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column",
        "height": 45,
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#329932",
          "flex": 0,
          "fontFamily": "telegrama_raw",
          "height": 19,
          "includeFontPadding": false,
          "lineHeight": 22,
          "textAlign": "center",
          "textAlignVertical": "bottom",
        }
      }
    >
      Actions: 
      10
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
          "height": 15,
          "justifyContent": "space-between",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          Character
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          [ Timelines ]
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          Feed (2)
        </Text>
      </View>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#329932",
          "fontSize": 45.911331934501746,
          "height": 40.019999999999996,
          "includeFontPadding": false,
          "lineHeight": 15,
          "textAlignVertical": "top",
        }
      }
    >
      +--------------------------------------------------------+
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 2,
      }
    }
  />
</View>
`;

exports[`Navbar (timeline-grid) => renders correctly 2`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#141313",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column",
        "height": 45,
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#329932",
          "flex": 0,
          "fontFamily": "telegrama_raw",
          "height": 19,
          "includeFontPadding": false,
          "lineHeight": 22,
          "textAlign": "center",
          "textAlignVertical": "bottom",
        }
      }
    >
      Actions: 
      10
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
          "height": 15,
          "justifyContent": "space-between",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          Character
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          Timelines
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flex": 1,
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#329932",
              "flex": 1,
              "fontFamily": "telegrama_raw",
              "fontSize": 13,
              "height": 9,
              "includeFontPadding": false,
              "lineHeight": 9,
              "textAlign": "center",
              "textAlignVertical": "bottom",
            }
          }
        >
          [ Feed (2) ]
        </Text>
      </View>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#329932",
          "fontSize": 45.911331934501746,
          "height": 40.019999999999996,
          "includeFontPadding": false,
          "lineHeight": 15,
          "textAlignVertical": "top",
        }
      }
    >
      +--------------------------------------------------------+
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 2,
      }
    }
  />
</View>
`;

exports[`Navbar (winner declared) => renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#141313",
      "flexDirection": "column",
      "height": "100%",
      "paddingBottom": 5,
      "paddingTop": 5,
      "width": "100%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#329932",
        "fontSize": 45.911331934501746,
        "height": 40.019999999999996,
        "includeFontPadding": false,
        "lineHeight": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
        "textAlignVertical": "top",
      }
    }
  >
    +------------------------------------------------------+
  </Text>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#329932",
          "flex": 0,
          "lineHeight": 13,
          "textAlignVertical": "top",
          "width": 22.5,
        }
      }
    >
       | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 

    </Text>
    <View
      style={
        Object {
          "backgroundColor": "#141313",
          "flexDirection": "column",
          "height": "100%",
          "maxWidth": 291,
          "paddingBottom": 5,
          "paddingTop": 5,
          "width": "100%",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "alignSelf": "center",
            "backgroundColor": "#141313",
            "color": "#329932",
            "flex": 1,
            "fontFamily": "telegrama_raw",
            "fontSize": 25,
            "textAlign": "center",
            "textAlignVertical": "center",
          }
        }
      >
        GAME OVER
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "alignSelf": "center",
            "backgroundColor": "#141313",
            "color": "#329932",
            "flex": 1,
            "fontFamily": "telegrama_raw",
            "fontSize": 25,
            "textAlign": "center",
            "textAlignVertical": "center",
          }
        }
      >
        Winner: 
        Aram
      </Text>
      <Image
        source={
          Object {
            "uri": "http://10.0.0.6:3000/placeholders/nopic.png",
          }
        }
        style={
          Object {
            "alignSelf": "center",
            "backgroundColor": "#141313",
            "borderColor": "black",
            "flex": 2,
            "marginBottom": 20,
            "maxHeight": 200,
            "maxWidth": 200,
            "resizeMode": "stretch",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "green",
              "height": "100%",
              "opacity": 0.5,
              "width": "100%",
            }
          }
        />
      </Image>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#329932",
          "flex": 0,
          "lineHeight": 13,
          "textAlignVertical": "top",
          "width": 22.5,
        }
      }
    >
       | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 

    </Text>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#329932",
        "fontSize": 45.911331934501746,
        "height": 40.019999999999996,
        "includeFontPadding": false,
        "lineHeight": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
        "textAlignVertical": "top",
      }
    }
  >
    +------------------------------------------------------+
  </Text>
</View>
`;
